apps/aecore/lib/aecore/oracle/oracle_state_tree.ex

```diff
@@ -192,7 +193,14 @@ defmodule Aecore.Oracle.OracleStateTree do
     case PatriciaMerkleTree.lookup(tree, key) do
       {:ok, serialized} ->
         {:ok, deserialized} = Serialization.rlp_decode_anything(serialized)
-        deserialized
+
+        case deserialized do
+          %Oracle{} ->
+            %{deserialized | owner: Identifier.create_identity(key, :oracle)}
```

Comment:
is the owner supposed to be `:oracle`?

Reply:
Yes, it supposed to be like that, same as `epoch`